proto/ibc/applications/middleware/fee/v1/genesis.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package ibc.applications.middleware.fee.v1;
+
+import "gogoproto/gogo.proto";
+
+// GenesisState defines the ibc-transfer genesis state
+message GenesisState {}
```

Comment:
Not too sure what state we want here @AdityaSripal

Reply:
mapping from packetID to escrowed fees

Reply:
what type are you thinking for escrowed fees? sdk.Coins?